operationsgateway_api/src/experiments/experiment.py

```diff
@@ -169,6 +197,31 @@ def _extract_experiment_data(
             except ValidationError as exc:
                 raise ModelError(str(exc)) from exc
 
+    def _get_mapping_model_by_experiment_id(
+        self,
+        experiment_id: int,
+        mapping_models: List[ExperimentPartMappingModel],
+    ):
+        """
+        Search through a list of experiment part mapping models and return the one with
+        a matching experiment ID
+        """
+
+        part_mapping = None
```

Comment:
Setting this variable and then checking for it on line 215 are not required. They can be removed and then the indentation of the remaining code adjusted without affecting any functionality.

Reply:
Fixed in 39a16f708d7b3579f201f6bc093f0172465699c8